mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
+ignore_missing_imports = True
```

Comment:
Which imports do we use that can't be resolved?

Reply:
3rd party libraries:
```
12: error: Cannot find implementation or library stub for module named 'structlog'  [import]
12: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
13: error: Cannot find implementation or library stub for module named 'structlog.stdlib'  [import]
14: error: Cannot find implementation or library stub for module named 'tenacity'  [import]
40: error: Cannot find implementation or library stub for module named 'urlparse'  [import]
```